modules/ROOT/pages/veevavault/veevavault-connector-design-center.adoc

```diff
@@ -0,0 +1,60 @@
+= VeevaVault Connector Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Design Center's Flow Designer enables you to create apps visually. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Flow Designer:
+
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
+
+For information on Flow Designer field values, see the xref:./veevavault-connector-reference.adoc[VeevaVault Connector Reference].
+
+
+== Configure the Input Source Trigger
+
+. In Design Center, click *Create*.
+. Click *Create new application* to open Flow Designer.
+. Specify a value for *Project name*, and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started*.
+. Click the trigger card.
+. If you are using the HTTP Listener operation as a trigger, search for and select HTTP Listener.
+. In the HTTP Listener’s General tab, enter the path-absolute-URL that defines the resource location.	
+. Click off the trigger card.
+
+Flow Designer automatically saves all changes.
+
+== Configure the Target Component
+
+The target component enables your app to process data received from the trigger. VeevaVault Connector provides operations that help you integrate VeevaVault with third party applications and services. For example, this integration enables you to creates vault connectivity and retrieve picklists data from Vault, which can be further integrated or process.
+
+. Click *+* next to the trigger card.
+. In *Select a component*, search for the Veeva Vault Connector.
+. Select the connector name.
+. Select an operation for the connector, such as *Get Picklists*.
+. If needed, specify values that are required in the General tab. For the Get Picklists operation, enter the required Picklist Name value whose details you want to retrieve.
+. Select Click here to set it up to set up other required fields.
+. Enter Configuration Name.
+. Click Add Connection and enter connection name.
+. In Connection Type, Select *1-Basic Authentication*.
+. In General tab, enter below required fields:
```

Comment:
```suggestion
. In the General tab, complete the following required fields:
```
This step could also be simplified to "Complete the General tab fields."

Reply:
If all the fields are required, as implied by the table intro., why does the table need a third column?
And if there are only two fields...consider a definition/description list.